contributed_definitions/NXsensor_scan.nxdl.xml

```diff
@@ -24,32 +24,32 @@
 <definition xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" category="application" type="group" name="NXsensor_scan" extends="NXobject" xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd">
```

Comment:
Please set the start date manually to the year 2022 as this was the initial year when we have first published it.